config.example.toml

```diff
@@ -33,6 +34,12 @@ usermap = "fedora_jira_usermap.toml"  # External file, can be reused
 # fedora_user1 = "jira_user1"
 # fedora_user2 = "jira_user2"
 
+# This section converts specific label on repository to story point
+[instances."pagure.io".story_points]
+label1 = 1
+label2 = 5
+label3 = 10
```

Comment:
How about more real life label names as examples, like `simple`, `complex`, `little-work`, `medium-work`, `lots-of-work`?

Reply:
Can do, that was just first that came to my mind.